pom.xml

```diff
@@ -34,11 +62,11 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.17</version>
         <configuration>
-          	<argLine>-Xmx2G -Xms2G -XX:MaxPermSize=512m</argLine>
```

Comment:
When I was running locally I got notices that MaxPermSize has been deprecated (or something to that effect) and is being ignored, so I thought might as well get rid of it.  Should I put it back in?